pkg/scheduler/framework/plugins/defaultpreemption/default_preemption.go

```diff
@@ -206,11 +227,11 @@ func (pl *DefaultPreemption) SelectVictimsOnNode(
 	if status := pl.fh.RunFilterPluginsWithNominatedPods(ctx, state, pod, nodeInfo); !status.IsSuccess() {
 		return nil, 0, status
 	}
-	var victims []*v1.Pod
+	var victims []*framework.PodInfo
```

Comment:
Why not keep the current implementation `[]*v1.Pod` ?

Reply:
I had this thinking:
- Further down here, the full PodInfo is being used for `removePod()` which then calls `fh.RunPrefilterExtensionRemovePod()`, so mapping to Pods and then back to PodInfos would be tricky.
- The provided OrderedPods function could have a use for other information in the PodInfo anyway